.github/workflows/on-prs.yml

```diff
@@ -0,0 +1,38 @@
+name: On Prs
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  approve:
+    if: contains(github.event.pull_request.labels.*.name, 'auto-approve-me')
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: hmarr/auto-approve-action@v3
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v3
+
+    - name: Setup Environment
+      uses: ./.github/actions/setup-env
+      with:
+        go: 'true'
+        go-cache: 'false'
+
+    - name: Lint
+      uses: golangci/golangci-lint-action@v6
+      with:
+        version: v1.57.1
```

Comment:
We agreed to update to v1.60.3. That said, I have a card for updating all of our workflows so if you want to punt and let me update that is fine. Though you should still consider updating your local linter version
```bash
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.60.3
```

Reply:
done